File: app/src/main/java/forpdateam/ru/forpda/api/regex/parser/Document.java
package forpdateam.ru.forpda.api.regex.parser;

/**
 * Created by radiationx on 13.08.17.
 */

public class Document extends Node {
    public final static String DOCTYPE_TAG = "!DOCTYPE";
    private String docType = "html";

    public Document() {
        super(NODE_DOCUMENT);
    }

    public String getDocType() {
        return docType;
    }

    public void setDocType(String docType) {
        this.docType = docType;
    }
}

File: app/src/main/java/forpdateam/ru/forpda/model/data/remote/api/regex/parser/Document.kt
package forpdateam.ru.forpda.model.data.remote.api.regex.parser

/**
 * Created by radiationx on 13.08.17.
 */
class Document : Node(NODE_DOCUMENT) {
    @JvmField
    var docType: String = "html"

    companion object {
        const val DOCTYPE_TAG: String = "!DOCTYPE"
    }
}
